projects/lib/src/interceptors/default-oauth.interceptor.ts

```diff
@@ -39,7 +47,7 @@ export class DefaultOAuthInterceptor implements HttpInterceptor {
         if (!this.moduleConfig.resourceServer) {
             return next.handle(req);
         }
```

Comment:
Maybe move the `moduleConfig.resourceServer` check into the `checkUrl` function as well then?

Reply:
do not think this is good idea because there can be other configuration in resourceServer that are used not only for checking urls in the future. Dont you think so/ So i propose to leave it where it is